src/app/pipes/propertydisplay.ts

```diff
@@ -34,7 +35,24 @@ export class PropertyValueDisplayPipe implements PipeTransform {
     const dtRegEx = /^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2})(?:\.\d+)?(?:Z|[-+]00(?::00)?)?$/;
     // Can't use typeof to determine if it's an Array
     if (value instanceof Array) {
-      return value.length.toString();
+      console.log(value);
+      if (value.length === 0) {
+        return '0';
+      } else {
+        if (value[0].advisory_name) {
```

Comment:
Adding a comment here explaining why we specifically check to see if this is an advisory would be nice.